### PR TITLE
RDAPI-23337 - Update March Release Note with details of upgrade to libthrift

### PR DESCRIPTION
**Pull request by credmond-axway**
Description
Libthrift has been upgraded due to known vulnerabilities. This PR updates the March release notes with details of the resolution.

Deploy Preview Link
[https://deploy-preview-1708--axway-open-docs.netlify.app/docs/apim_relnotes/20210330_apimgr_relnotes/#fixed-security-vulnerabilities](https://deploy-preview-1708--axway-open-docs.netlify.app/docs/apim_relnotes/20210330_apimgr_relnotes/#fixed-security-vulnerabilities)

### CODE REVIEW

**Comment by credmond-axway** [comment]
@JuanAxway @kmercierwhite

Please review.

Preview available here: https://deploy-preview-1708--axway-open-docs.netlify.app/docs/apim_relnotes/20210330_apimgr_relnotes/#fixed-security-vulnerabilities

**Review by JuanAxway — APPROVED** [review]

File: content/en/docs/apim_relnotes/20210330_apimgr_relnotes.md [hunk 1]
```diff
@@ -79,7 +79,9 @@ This version of API Gateway and API Manager includes:
...
+| ----------- | ------------------ | -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| RDAPI-23337 | | CVE-2018-1320 CVE-2019-0205 | **Issue**: Apache Cassandra includes libthrift library which has known vulnerabilities. **Resolution**: New installations which opt to install Cassandra from the API Gateway installation packages will include the upgraded libthrift 0.9.3-1 library that addresses known vulnerabilities. Existing customers are advised to upgrade the libthrift library to 0.9.3-1 in their Cassandra environments.                                                                                                         |
```
Comment (mdenihan) [inline content/en/docs/apim_relnotes/20210330_apimgr_relnotes.md:84]: I'd just change the wording slightly;

> 0.9.3-1 library that addresses **these two** known vulnerabilities

**Review by mdenihan — CHANGES_REQUESTED** [review]

File: content/en/docs/apim_relnotes/20210330_apimgr_relnotes.md (hunk 1 above)
Comment (credmond-axway) [reply content/en/docs/apim_relnotes/20210330_apimgr_relnotes.md]: Thanks @mdenihan. I've updated the wording.

**Review by mdenihan — APPROVED** [review]
Thanks @credmond-axway

**Comment by andreamussap** [comment]
@cpalmeraxway Please attention to this PR so you don't overwrite the issue in this PR adding when you create your PR to add the list of fixed issues. Thanks.

**Review by andreamussap — APPROVED** [review]

**Pull request merged**